.gitignore

```diff
@@ -1,3 +1,4 @@
+/.pdm-python
```

Comment:
No…and yes.

I'm sporadically working on a PR to use a Python venv using PDM as the package manager for the Python tasks in chezmoi development. It has a number of really nice advantages over the current `requirements.txt`. It isn't *quite* ready, but every time I switch branches this shows up as unmanaged, and I got annoyed this time and added it to `.gitignore`.

It's actually part of the "since-version" chip that I was talking about, but I should probably extract the PDM part and open it as a PR on its own.

I can remove that from this PR if you wish.

Reply:
Please do remove it from this PR -- you can add `.pdm-python` to your global `.gitignore` file in the short term and add it to chezmoi's `.gitignore` file with your PDM PR :)